atd-vze/src/Components/DataTable.js

```diff
@@ -26,9 +35,18 @@ const DataTable = ({
     setEditField("");
   };
 
+  const onButtonClick = (e, section) => {
+    handleFieldUpdate && handleFieldUpdate(e, section, "button");
+    handleButtonClick &&
+      handleButtonClick(e, section.button.buttonFieldUpdate, data);
+  };
+
   return (
     <>
       {dataMap.map((section, i) => {
+        const buttonCondition =
+          section.button && section.button.buttonCondition;
```

Comment:
After speaking with Mike we decided to remove the extra `section.button.buttonCondition` statement that was present at a previous commit.